pyhap/service.py

```diff
@@ -50,21 +45,26 @@ def get_characteristic(self, name):
                 return char
         raise ValueError('Characteristic not found')
 
-    def to_HAP(self, iid_manager=None):
-        """Create a HAP representation of this Service.
+    def configure_char(char_name, properties=None, valid_values=None,
+                       value=None, callback=None):
```

Comment:
can we change it to setter_callback, as we expect to have a getter_callback as well?